tests/epyccel/modules/blas.py

```diff
@@ -0,0 +1,3317 @@
+        lda = k
+        ldb = k
+    # ...
+
+    # ...
+    flag_uplo = 'U'
+    if lower : flag_uplo = 'L'
+    # ...
+
+    ldc = n
+
+    mod_blas.ssyr2k (flag_uplo, flag_trans, n, k, alpha, a, lda, b, ldb, beta, c, ldc)
+
+# ==============================================================================
+def blas_strmm(alpha: 'float32', a: 'float32[:,:](order=F)', b: 'float32[:,:](order=F)',
+               side: 'bool' = False,
+               lower: 'bool' = False,
+               trans_a: 'bool' = False,
+               diag: 'bool' = False
+              ):
+
+    import numpy as np
+    import pyccel.stdlib.internal.blas as mod_blas
+
+    m = np.int32(b.shape[0])
+    n = np.int32(b.shape[1])
+
+    # ...
+    # equation 1
+    flag_side = 'L'
+    lda = m
+    # equation 2
+    if side:
+        flag_side = 'R'
+        lda = n
+    # ...
+
+    # ...
+    flag_uplo = 'U'
+    if lower : flag_uplo = 'L'
+    # ...
+
+    # ...
+    flag_trans_a = 'N'
+    if trans_a: flag_trans_a = 'T'
+    # ...
+
+    # ...
+    flag_diag = 'N'
+    if diag: flag_diag = 'U'
+    # ...
+
+    ldb = m
+
+    mod_blas.strmm (flag_side, flag_uplo, flag_trans_a, flag_diag, m, n, alpha, a, lda, b, ldb)
+
+# ==============================================================================
+def blas_strsm(alpha: 'float32', a: 'float32[:,:](order=F)', b: 'float32[:,:](order=F)',
+               side: 'bool' = False,
+               lower: 'bool' = False,
+               trans_a: 'bool' = False,
+               diag: 'bool' = False
+              ):
+
+    import numpy as np
+    import pyccel.stdlib.internal.blas as mod_blas
+
+    m = np.int32(b.shape[0])
+    n = np.int32(b.shape[1])
+
+    # ...
+    # equation 1
+    flag_side = 'L'
+    lda = m
+    # equation 2
+    if side:
+        flag_side = 'R'
+        lda = n
+    # ...
+
+    # ...
+    flag_uplo = 'U'
+    if lower : flag_uplo = 'L'
+    # ...
+
+    # ...
+    flag_trans_a = 'N'
+    if trans_a: flag_trans_a = 'T'
+    # ...
+
+    # ...
+    flag_diag = 'N'
+    if diag: flag_diag = 'U'
+    # ...
+
+    ldb = m
+
+    mod_blas.strsm (flag_side, flag_uplo, flag_trans_a, flag_diag, m, n, alpha, a, lda, b, ldb)
+
+# ==============================================================================
+#
+#                                  LEVEL 1
+#
+# ==============================================================================
+
+# ==============================================================================
+def blas_drotg(a: 'float64', b: 'float64',
+               c: 'float64' = 0.,
+               s: 'float64' = 0.,
+              ):
+
+    import numpy as np
+    import pyccel.stdlib.internal.blas as mod_blas
+
+    mod_blas.drotg (a, b, c, s)
+
+    return c, s
+
+# ==============================================================================
+def blas_drotmg(d1: 'float64', d2: 'float64', x1: 'float64', y1: 'float64',
+                param: 'float64[:]'):
+
+    import numpy as np
```

Comment:
There are lots of unused `import numpy as np` statements in this file. They should be flagged by codacy when it starts working again